### DISCUSSION TITLE
Bring back MSN

### DISCUSSION THREAD

**Discussion by Selfieh** [ppy/osu#30213] [Ideas] [answered]
Hey there, i enjoy moving from stable to lazer and all the new features. but i'm missing a big feature that was present in stable. MSN integration. i stream my gameplay on twitch from time to time and have a !np command for people to get the link to the beatmap that i have selected or that i'm currently playing. therefore i use Stream Companion to get the link. but it doesnt work in lazer obviously because stream companion stopped working. so i would like to ask for a little feature that can make the swap from stable to lazer easier. i did a little research but didnt find any alternatives or solutions to this problem in lazer. maybe a feature in the settings for exporting beatmap data to a file (let users decide which data goes into what file) or anything that can solve this problem at all.

**Comment by bdach** [comment, marked as answer]
MSN is not coming back. Your hopes should be in something like https://github.com/ppy/osu/pull/18129.
